sdk/src/main/scala/com/horizen/account/receipt/EvmLogUtils.scala

```diff
@@ -50,7 +50,12 @@ object EvmLogUtils extends ScorexSerializer[EvmLog] {
       rlpTopics.add(RlpString.create(t.toBytes))
     }
     result.add(new RlpList(rlpTopics))
-    result.add(RlpString.create(log.data))
+    if (log.data == null){
+      result.add(RlpString.create(Array[Byte](0)))
+    }
+    else {
+      result.add(RlpString.create(log.data))
+    }
```

Comment:
do we need it?
let's ensure that `log.data == null` is never a case for us.

Reply:
I need to check if JSon parser can initialize an EvmLog object with private fields. If it can, I can force data to be != null.